.github/workflows/main_workflow.yml

```diff
@@ -284,26 +284,26 @@ jobs:
           # docker pull trufflesuite/ganache:beta
 
           # build hardhat node in third_party/safe-contracts
-          cd third_party/safe-contracts &&\
-            sudo chown -R $(whoami) .husky &&\
-            yarn
-          cd ../..
-
-          # build hardhat node in third_party/contracts-amm
-          cd third_party/contracts-amm &&\
-            yarn
-          cd ../..
+          # cd third_party/safe-contracts &&\
```

Comment:
Keeping it here for ease of reference, but we don't need it for mac and windows!